packages/components/addon/components/hds/table/th.hbs

```diff
@@ -6,6 +6,15 @@
   {{#if @isVisuallyHidden}}
     <span class="sr-only">{{yield}}</span>
   {{else}}
-    {{yield}}
+    {{#if @tooltip}}
+      <div class="hds-table__th-content">
+        {{#let (unique-id) as |labelId|}}
+          <span id={{labelId}} class="hds-typography-body-200 hds-font-weight-semibold">{{yield}}</span>
+          <Hds::Table::ThButtonTooltip @tooltip={{@tooltip}} @labelId={{labelId}} />
+        {{/let}}
+      </div>
+    {{else}}
+      <span class="hds-typography-body-200 hds-font-weight-semibold">{{yield}}</span>
```

Comment:
If we want, we can remove this duplication by moving the `{{#if @tooltip}}` check just before the `<Hds::Table::ThButtonTooltip>` element (but in this case the table cell content will always have a  `<div class="hds-table__th-content">` wrapper. WDYT?

Reply:
I don't mind having it as it is, the duplication is minimal and keeps the mark-up to a minimum.